letsencrypt/cli.py

```diff
@@ -889,15 +976,24 @@ def prepare_and_parse_args(plugins, args):
         version="%(prog)s {0}".format(letsencrypt.__version__),
         help="show program's version number and exit")
     helpful.add(
+        "automation", "--expand", "--expand-existing-certs", "--replace", action="store_true",
+        help="If an existing cert covers some subset of the requested names, "
+             "expand and replace it with the additional names.")
+    helpful.add(
         "automation", "--renew-by-default", action="store_true",
         help="Select renewal by default when domains are a superset of a "
-             "previously attained cert")
+             "previously attained cert (often --keep-until-expiring is "
+             "more appropriate). Implies --expand.")
     helpful.add(
         "automation", "--agree-tos", dest="tos", action="store_true",
         help="Agree to the Let's Encrypt Subscriber Agreement")
     helpful.add(
         "automation", "--account", metavar="ACCOUNT_ID",
         help="Account ID to use")
+    helpful.add(
+        "automation", "--duplicate", dest="duplicate", action="store_true",
```

Comment:
`HelpfulArgumentParser` doesn't support this, but perhaps we should make some of these `automation` flags mutually exclusive. It makes the code and interaction between the flags simpler.